gitops/argocd/tektoncd/kustomization.yaml

```diff
@@ -2,4 +2,3 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - openshift-operator.yaml
-  - tekton-config.yaml
```

Comment:
Better to comment this line out and include a comment mentioning the reason for it (as compared to deleting the line). Even if another dev picks up this issue, they will have context.

Reply:
Ok sure